dashboard/pages/primary_screening/callbacks.py

```diff
@@ -217,6 +223,102 @@ def upload_echo_data(contents, names, last_modified, stored_uuid, file_storage):
 # === STAGE 5 ===
 
 
+def on_stage_5_entry(current_stage: int, stored_uuid: str, file_storage: FileStorage):
```

Comment:
suggestion: use explicit stage name instead of index in the callback name - it makes it more clear what should be in the callback and in general what it refers to, and the index itself may be inferred from the `if current_stage` check

Reply:
Good point, will do that